configure.ac

```diff
@@ -685,6 +685,24 @@ AS_IF([test "$have_synapseai" = "1" && test x"$with_synaposeai" != x"yes"],
       [CPPFLAGS="$CPPFLAGS $synapseai_CPPFLAGS"
        LDFLAGS="$LDFLAGS $synapseai_LDFLAGS"])
 
+dnl Check for LTTNG tracking capability
+have_lttng=0
+AC_ARG_WITH([lttng],
+    AS_HELP_STRING([--with-lttng=DIR],
+		   [Enable tracing capability with LTTNG @<:@default=no@:>@]))
+
+if test "$with_lttng" != "" && test "$with_lttng" != "no"; then
+	AC_DEFINE([INCLUDE_LTTNG], 1,
+		  [Define to 1 to enable tracing capability with lttng])
+	if test -d $with_lttng; then
```

Comment:
use `FI_CHECK_PACKAGE` to check the existence of lttng library

Reply:
Update the commit message to reference configure or build or core, and remove the reference to prov/efa, as these changes are not limited to efa.